src/main/kotlin/no/nav/klage/oppgave/service/KlagebehandlingService.kt

```diff
@@ -97,6 +110,10 @@ class KlagebehandlingService(
         utfoerendeSaksbehandlerIdent: String
     ): Klagebehandling {
         val klagebehandling = getKlagebehandlingForUpdate(klagebehandlingId, klagebehandlingVersjon, true)
+        if (tildeltSaksbehandlerIdent != null) {
+            //Dette er en assignment!
```

Comment:
Hva betyr dette?

Reply:
Samme metode brukes for å tildele og å fradele, og det gir ikke meningen å sjekke tilgangene for fradeling, for det er tilgangene til den man assigner til som blir sjekket

Reply:
Har endret kommentaren